content/docs/quickstart/cpp.md

```diff
@@ -9,63 +9,111 @@ description: This guide gets you started with gRPC in C++ with a simple working
 
 #### gRPC
 
-To install gRPC on your system, follow the [instructions to install gRPC C++ via make](https://github.com/grpc/grpc/blob/master/src/cpp/README.md#make).
+Follow the gRPC installation instructions for your OS:
 
-To run the example code, please ensure `pkg-config` is installed on your
-machine before you build and install gRPC in the previous step, since the
-example `Makefile`s try to look up the installed gRPC path using `pkg-config`.
-On Debian-based systems like Ubuntu, this can usually be done via
-`sudo apt-get install pkg-config`.
+- macOS:
+
+  ```sh
+  $ brew install grpc
+  ```
+
+- Any OS: to build and install from sources, refer to [Building from
+  source][BUILDING].
 
 #### Protocol Buffers v3
 
-While not mandatory to use gRPC, gRPC applications usually leverage Protocol
-Buffers v3 for service definitions and data serialization, and our example code
-uses Protocol Buffers as well as gRPC. If you don't already have it installed on
-your system, you can install the version cloned alongside gRPC. First ensure
-that you are running these commands in the gRPC tree you just built in the from
-the previous step.
+While not mandatory, gRPC applications usually leverage [Protocol Buffers
+v3][pbv3] for service definitions and data serialization, and the example code
+uses Protocol Buffers.
+
+- macOS (if you installed `grpc` using brew, then you can skip this step since
+  `protobuf` is installed as a prerequisite):
+
+  ```sh
+  $ brew install protobuf
+  ```
+
+- Any OS:
+
+   1. Download a zip file of the latest version of pre-compiled binaries for
+      your operating system from [github.com/google/protobuf/releases][]
+      (`protoc-<version>-<os>.zip`).
+   2. Unzip the file.
+   3. Update your environment's path variable to include the path to the
```

Comment:
while getting the protoc binaries that way is straighforward and useful, it won't really help you build a simple grpc application because you'll also need libprotobuf, libprotoc and the gRPC protoc plugins to be able to build your example app.
And once you're able to get libprotobuf built and installed, you can trivially build and install the protoc binaries.

Reply:
I've tested this approach under macOS and it worked fine for me, as far as I recall. I agree that this page will require further edits, but I seen this as I first round of changes. Can we keep this text as is for now?

If you prefer that I remove these lines I can. I'd just like to get the remaining edits is so that we can start another pass.

Reply:
Removed